### PR TITLE
Fix Search Bar header for Staff

### PR DESCRIPTION
**Pull request by dimak1**
*Description of changes:*

- Check `mhr-ui-enabled` flag for the Search Bar header

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/views/Dashboard.vue [hunk 1]
```diff
@@ -30,9 +30,10 @@
                   id="search-header"
                   :class="[$style['dashboard-title'], 'pl-6', 'pt-3', 'pb-3', 'soft-corners-top']">
             <v-col cols="auto">
-              <b v-if="hasPPRProduct && hasMHRProduct">Manufactured Home and Personal Property Registry Search</b>
-              <b v-else-if="hasPPRProduct">Personal Property Search</b>
-              <b v-else-if="hasMHRProduct">Manufactured Home Search</b>
+              <b v-if="hasPPR && hasMHR">
+                Manufactured Home and Personal Property Registry Search</b>
```
Comment (cameron-eyds) [inline ppr-ui/src/views/Dashboard.vue:34]: too many chars for 1 line?

File: ppr-ui/src/views/Dashboard.vue [hunk 2]
```diff
@@ -431,12 +437,22 @@ export default class Dashboard extends Vue {
     return this.getSearchHistory?.length || 0
   }
 
-  private get hasPPRProduct (): boolean {
-    return this.getUserProductSubscriptionsCodes.includes(ProductCode.PPR)
+  private get hasPPR (): boolean {
+    // For Staff we check roles, for Client we check Products
+    if (this.isRoleStaff || this.isRoleStaffBcol || this.isRoleStaffReg) {
```
Comment (cameron-eyds) [inline ppr-ui/src/views/Dashboard.vue:442]: For a consolidated staff check like this, is there already a getter? If not, perhaps we could create one.
Seems useful, saves some imports and duplication here if created in the store.

**Review by cameron-eyds — APPROVED** [review]
Looks good, just 1 suggestion!

**Review by doug-lovett — APPROVED** [review]

File: ppr-ui/src/views/Dashboard.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/views/Dashboard.vue:34]: Yea, had to break it

File: ppr-ui/src/views/Dashboard.vue (hunk 2 above)
Comment (dimak1) [reply ppr-ui/src/views/Dashboard.vue:442]: There is no generic getter to check for all staff options. I could add one, as we use it in multiple places.

Comment (dimak1) [reply ppr-ui/src/views/Dashboard.vue:442]: I will add this getter in another PR. Just want to keep this one as a small fix.

**Pull request merged**